Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift

```diff
@@ -43,26 +79,44 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
       return
     }
     let eventProperties = eventPropertiesSection.eventProperties()
-    if (MSTransmissionTargets.shared.defaultTargetShouldSendAnalyticsEvents()) {
-      if let properties = eventProperties as? MSEventProperties {
-
-        // The AppCenterDelegate uses the argument label "withTypedProperties," but the underlying swift API simply uses "withProperties."
-        appCenter.trackEvent(name, withTypedProperties: properties)
-      } else if let dictionary = eventProperties as? [String: String] {
-        appCenter.trackEvent(name, withProperties: dictionary)
-      } else {
-        appCenter.trackEvent(name)
-      }
-    }
-    for targetToken in MSTransmissionTargets.shared.transmissionTargets.keys {
-      if MSTransmissionTargets.shared.targetShouldSendAnalyticsEvents(targetToken: targetToken) {
-        let target = MSTransmissionTargets.shared.transmissionTargets[targetToken]
+    for _ in 0..<Int(countSlider.value) {
+      if (MSTransmissionTargets.shared.defaultTargetShouldSendAnalyticsEvents()) {
         if let properties = eventProperties as? MSEventProperties {
-          target!.trackEvent(name, withProperties: properties)
+
+          // The AppCenterDelegate uses the argument label "withTypedProperties", but the underlying swift API simply uses "withProperties".
+          if priority != .Default {
+            appCenter.trackEvent(name, withTypedProperties: properties, flags: priority.flags)
+          } else {
+            appCenter.trackEvent(name, withTypedProperties: properties)
+          }
         } else if let dictionary = eventProperties as? [String: String] {
-          target!.trackEvent(name, withProperties: dictionary)
+          if priority != .Default {
+            appCenter.trackEvent(name, withProperties: dictionary, flags: priority.flags)
+          } else {
+            appCenter.trackEvent(name, withProperties: dictionary)
+          }
         } else {
-          target!.trackEvent(name)
+          appCenter.trackEvent(name)
```

Comment:
We should set a priority in this case.

Reply:
Android checks if not using default flag in UI then we call trackEvent(name, null, flags) and trackEvent(name) otherwise, should indeed do the same.